redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+  Container,
+  Section,
+  Hero,
+  Columns,
+  Heading,
+  Content,
+} from "react-bulma-components";
+import Button from "../atoms/Button";
+import team from "../../assets/images/hero.svg";
+import teamMobile from "../../assets/images/hero-mobile.svg";
+import "./RediHero.scss";
+
+const RediHero = () => (
+  <Hero>
+    <Container>
+      <Section>
+        <Columns>
+          <Columns.Column size={5} className="hero__column">
+            <Heading size={1} renderAs="h1" className="hero__column--heading">
```

Comment:
here as well... check out the documentation of http://getbem.com/introduction/ if you are interested.
I am however not sure if it really make sense to use it in this project because the it is not used with Bulma... so it's a bit strange to se here and there some bem...

Reply:
I changed the classNames within the component to single-dash logic like Bulma does it. Maybe as an idea: whenever we create our own components we use BEM, if it's mainly Bulma we use their notation?